frontend/packages/core/src/button.tsx

```diff
@@ -144,20 +160,23 @@ const variantPalette = (variant: ButtonVariant): ButtonPalette => {
 };
 
 export interface ButtonProps
-  extends Pick<MuiButtonProps, "disabled" | "endIcon" | "onClick" | "startIcon" | "type"> {
+  extends Pick<
+    MuiButtonProps,
+    "id" | "disabled" | "endIcon" | "onClick" | "startIcon" | "type" | "size"
```

Comment:
curious why `id` is picked

Reply:
To target the button in the NPS Banner component so that I could detect whether it was being pressed or somewhere in the body and cancel the normal close action that the Header uses